### PR TITLE
Qucs-activefilter technical description

### PR DESCRIPTION
**Pull request by ra3xdh**
I have added qucs-activefilter technical report. The paper contains mathematical background of active filters design and active filters circuitry description. Described methods are used inside Qucs-activefilter.

Compile it with LaTeX and see `qucs-doc/report/activefitler/activefilter.pdf`

### CODE REVIEW

File: qucs-doc/report/activefilter/Makefile.am [hunk 1]
```diff
@@ -0,0 +1,64 @@
+## Process this file with automake to produce Makefile.in
+# 
+# report/mod_amp/Makefile.am
```
Comment (guitorri) [inline qucs-doc/report/activefilter/Makefile.am]: Can you fix this, just for consistency.

File: qucs-doc/report/activefilter/Makefile.am [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+#
+# Automake input file.
+#
+# Copyright (C) 2007 Stefan Jahn <stefan@lkcc.org>
```
Comment (guitorri) [inline qucs-doc/report/activefilter/Makefile.am]: What about you? :)

**Comment by ra3xdh** [comment]
Corrected

**Comment by guitorri** [comment]
Perhaps you forgot to add the pics/Makefile.am?
`configure.ac:147: error: required file 'report/activefilter/pics/Makefile.in' not found`

**Comment by ra3xdh** [comment]
Added pics/Makefile.am

File: qucs-doc/report/activefilter/activefilter.tex [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+\include{titlepage}
+
+\tuttitle{
+  Report}
+\tutsubtitle{
+  Qucsactivefilter --- Active filter synthesis subsystem of Qucs }
+\tutauthor{
+  Vadim Kuznetsov}
+\tutcopyright{
+  Copyright \copyright{} 2014 Mike Brinson
```
Comment (guitorri) [inline qucs-doc/report/activefilter/activefilter.tex]: This should be you I guess.

File: qucs-doc/report/activefilter/pics/Makefile.am [hunk 4]
```diff
@@ -0,0 +1,38 @@
+## Process this file with automake to produce Makefile.in
+# 
+# report/StoQ_test1/pics/Makefile.am
```
Comment (guitorri) [inline qucs-doc/report/activefilter/pics/Makefile.am]: Here?

**Comment by guitorri** [comment]
Very nice, I trust you on the math :smile: 
Perhaps you could replace the `(uF)` by `[ \mu F]` to make clear you are talking about the capacitance unit, in micro-Farad. Just a suggestion.

**Comment by yodalee** [comment]
Not related to document but the building system.
I run into something like this when I run `autogen.sh` in qucs-doc:
```
Creating aclocal.m4... done.
Creating Makefile.in(s)... done.
Creating configure... done.
Running ./configure $ac_cs_config  ...
configure: WARNING: you should use --build, --host, --target
configure: WARNING: invalid host type: $ac_cs_config
checking build system type... Invalid configuration `$ac_cs_config': machine `$ac_cs_config' not recognized
configure: error: /bin/sh ./config.sub $ac_cs_config failed
yodalee@yodaleeNB qucs-doc$                                   
```
I believe the `$@` is the problem.

**Comment by ra3xdh** [comment]
Makefile.am  and activefilter.tex headers corrected. uF units corrected.

**Comment by yodalee** [comment]
I add some... maybe a lot ... comments after I print out the doc and read it. Great detailed document.

**Comment by ra3xdh** [comment]
Typos corrected. Duplicates removed. Here is explanations for other comments. See the last commit.

>  These two sentences looks weird. First you say find RC values by Ai, Bi, Ci. Second sentence you 
> said Ai, Bi, Ci is just a intermediate, which is found by zeros and poles.

Ai, Bi, Ci can be found from poles and zeros. RC-elements can be found from Ai, Bi, Ci.

>  There is no epsilon in equation {cheby2-order}, so R_p is not required. 
> Though I think it's a equation error.

There was text error.

> Just ask: why here is `$N-th$` and line 276 is `$i$-th` similar problem at line 411, line 524, line 620

There is no error Active filter contains only one first-order section or not contain it. Single first order section correspond only one Nth real pole and only single coefficient $C_N$


> I don't understand this. This mu seems not appear in discussion before.
> Why we can arbitrary set it to 2.0? If it's a constant then we can write 2.0 directly. 
> If it's a variable then what's the function of this variable? Also in line 1747

Added citation. This coefficient depends on quality factor. It is trimmed coefficient. 

> R2 -> R_2

There is no error. This is circuit components RefDes. Both designations are equivalent. You can find both in literature.

Other typos corrected.

**Comment by yodalee** [comment]
I got no question now.

**Comment by guitorri** [comment]
@ra3xdh shall I merge?

**Comment by ra3xdh** [comment]
Yes, all typos is now fixed. You can merge.

**Comment by guitorri** [comment]
Merged in 59913e19d80. Thank you.
Also added a reminder for the next time I update the website: https://github.com/Qucs/qucs.github.io/issues/3

**Pull request closed without merging**